app/src/main/java/org/oppia/app/options/StoryTextSizeFragment.kt

```diff
@@ -0,0 +1,45 @@
+package org.oppia.app.options
+
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import org.oppia.app.fragment.InjectableFragment
+import javax.inject.Inject
+
+private const val KEY_STORY_TEXT_SIZE_PREFERENCE_SUMMARY_VALUE =
+  "STORY_TEXT_SIZE_PREFERENCE_SUMMARY_VALUE"
+
+/** The fragment to change the Text size of the Story content in the app. */
```

Comment:
/** The fragment to change the text size of the story content in the app. */

Reply:
done